tests/functional/test_lists.py

```diff
@@ -0,0 +1,15 @@
+import pytest
+
+from pages.article import ArticlePage
+from selenium.webdriver.common.by import By
+
+
+@pytest.mark.nondestructive
+def test_list_macros(base_url, selenium):
```

Comment:
I'm wondering if we want to more fully check the functionality of this page, like adding assertions and/or click-tests for the links (macro-name, macro-counts) and the manual search buttons?